### ISSUE TITLE
Question:  Relative Attachment Folder Path?

### ISSUE THREAD

**Issue by jahlpub** [alberti42/obsidian-import-attachments-plus#17]
Hi!

How do you set the Attachment Folder Path to be relative to the location of the note?  For example, ./attachments/${notename} (attachments).

Thanks!

**Comment by alberti42** [comment]
Have you tried the following:

![Bildschirmfoto 2024-12-28 um 17 06 48](https://github.com/user-attachments/assets/2996d480-5f20-4c3d-bd0f-8c15d0793fdc)

**Comment by jahlpub** [comment]
thank you, i will try. On Dec 28, 2024, at 8:07 AM, Andrea Alberti ***@***.***> wrote:﻿
Have you tried the following:
Bildschirmfoto.2024-12-28.um.17.06.48.jpg (view on web)

**Comment by jahlpub** [comment]
hi, that worked fabulously!  thank you. on iphone mobile under Settings, i have all settings for Managing, no settings for  Importing nor Opening nor Attachments sections and only Hide Attachment Folders for Attachment folder section.   Would you please elaborate on the functionality i should expect?thanks!jayOn Dec 28, 2024, at 8:11 AM, J Lim ***@***.***> wrote:﻿thank you, i will try. On Dec 28, 2024, at 8:07 AM, Andrea Alberti ***@***.***> wrote:﻿
Have you tried the following:
Bildschirmfoto.2024-12-28.um.17.06.48.jpg (view on web)

**Comment by alberti42** [comment]
Yes, the ability to import documents is not available on mobile phones. On iPhone and iPad it is certainly not possible without jailbreaking the phone. On Androids, I do not know, but I have not implemented the functionality of importing by dragging documents. This only works for desktops platforms.

However, all the other features relating to extra buttons to remove attachments and hide attachment folders, and other nice attachment management options, should be available across platforms.

In short, the mobile version lacks the ability to move documents to the vault. But the rest should be there and usable.
